pkg/chunk/series_store.go

```diff
@@ -112,12 +143,19 @@ func (c *seriesStore) lookupSeriesByMetricNameMatchers(ctx context.Context, from
 
 	// Just get series for metric if there are no matchers
 	if len(matchers) == 0 {
-		return c.lookupSeriesByMetricNameMatcher(ctx, from, through, metricName, nil)
+		indexLookupsPerQuery.Observe(float64(1))
```

Comment:
You don't need to cast to `float64`. And if there is no `+1` below, then this would cause `0` lookups right?